data/sstable/md-2-big-Digest.crc32

```diff
@@ -0,0 +1 @@
+465310271
```

Comment:
so all these files will be present wherever a table is?

Reply:
Yes, each table should have all these 8 files generated by Cassandra on different folder